test/tools/router_check/router_check.cc

```diff
@@ -4,20 +4,39 @@
 
 #include "test/tools/router_check/router.h"
 
+bool isArgument(int argc, char* argv[], const std::string& argument) {
```

Comment:
Can you add a TODO above here to switch this to using TCLAP in a future change. We shouldn't be messing directly with argv and argc like this.